pkg/endpoints/cluster.go

```diff
@@ -162,3 +164,70 @@ func (r Resource) GetProperties(request *restful.Request, response *restful.Resp
 	properties := Properties{InstallNamespace: os.Getenv("INSTALLED_NAMESPACE")}
 	response.WriteEntity(properties)
 }
+
+// Get dashboard version
+func (r Resource) GetDashboardVersion(request *restful.Request, response *restful.Response) {
+	config, err := rest.InClusterConfig()
+	clientset, err := kubernetes.NewForConfig(config)
+	if err != nil {
+		panic(err.Error())
+	}
+
+	properties := Properties{InstallNamespace: os.Getenv("INSTALLED_NAMESPACE")}
+	version := ""
+
+	api1 := clientset.AppsV1()
+	listOptions := metav1.ListOptions{
+		LabelSelector: "app=tekton-dashboard",
+	}
+	deployments, err := api1.Deployments(properties.InstallNamespace).List(listOptions)
+
+	for _, deployment := range deployments.Items {
+		deploymentLabels := deployment.GetLabels()
+		version = deploymentLabels["version"]
+	}
+
+	if version == "" {
+		response.WriteEntity("Unknown")
+		return
```

Comment:
We could return an error here and catch on the frontend, displaying some meaningful warning (instead of a normal return which is a http code 200)

Reply:
I have put errors in if cant get the deployment 
I have put errors in if it is unknown and it returns a 500 
It however doesnt display any error on the dashboard yet that says there was an issue, the logs will say there was an issue